pages/blog/radical-transparency.md

```diff
@@ -0,0 +1,71 @@
+
+## What is Radical Transparency?
+
+> **radical**
+> /ˈradɪk(ə)l/
+>
+> (especially of change or action) relating to or affecting the fundamental nature of something; far-reaching or thorough. **Forming an inherent or fundamental part of the nature of someone or something.**
+
+> **transparency**
+> /tranˈsparənsi,trɑːnˈsparənsi/
+>
+> (of an organization or its activities) the condition of being **open to public scrutiny**.
+
+Radical transparency is about doing every single thing in public. It's about empowering the community with the necessary tools and information to scrutinize everything we do.
+
+## What does it mean for me as the Executive Director?
+
+Believe it or not, it's relieving. Most people —including myself in the past— would have been scared of letting so much power go and opt for radical transparency.
+
+I often get a lot of pressure from different companies to do something for them. Here are some examples:
+
+* Invite someone to the TSC in exchange for a [Platinum Sponsorship](https://opencollective.com/asyncapi/contribute/platinum-sponsors-8119).
+* Exchange logos on our websites. Oh my! It just happened as I'm writing these lines :joy:
+* Donate some money only if they get "something" in return. These people don't get open-source, do they? :smile:
+
+So Radical Transparency is actually making things easier for me. I just have to say something like this:
+
+> I'm sorry, but I can't do what you're suggesting. If you're still interested in a similar deal, please head over https://www.github.com/asyncapi/community and open an issue with your proposal. We can happily discuss it there.
+
+All of a sudden, it's not just my responsibility. It's [the TSC](https://www.asyncapi.com/community/tsc) who gets to decide if their proposal moves forward or doesn't. This makes the TSC (including myself) accountable for the decision. And most important, it's all registered and accessible to anyone in the community to scrutinize.
+
+Radical Transparency is removing all these trouble-makers for me. Whoever is truly interested in making a honest proposal, will also suggest it publicly. This system allows me to get rid of those whose interest is not making the project better and are only seeking profit. And, so far, these people stand for 100% of the cases.
+
+Of course, I'm still happy to take private calls and emails when they make sense. E.g. they contain sensitive information, you don't speak English, you're shy to propose things in public, they're not related to AsyncAPI, etc.
+
+## What does it mean for the community?
+
+Whether you are a user, a contributor, or a code owner, your decisions must be taken in public. When it's about code, we use pull requests. When it's something else, use issues or discussions on the related repo. And, if there's no related repo, use [the community one](https://www.github.com/asyncapi/community).
```

Comment:
```suggestion
Whether you are a user, a contributor, or a code owner, everyone must make decisions in public. For code, we use pull requests. Use issues or discussions on the related repo when it's something else. And, if there's no related repo, use [the community one](https://www.github.com/asyncapi/community).  
```

Wanted to make a grammar suggestion only, but this section is confusing to me. First, it seems to indicate that we only use PRs for code, but we also use them for things such as voting for TSC items and docs... And people create issues before creating PRs, so when you write "use issues for something else," it sounded incorrect or not very clear to me. Just a thought 😀

Reply:
Yeah, looks much better now. Thanks 🙏